hc/front/views.py

```diff
@@ -186,6 +188,59 @@ def update_nag_interval(request, code):
     return redirect("hc-checks")
 
 
+@login_required
+def shopify_alerts(request, code):
+    assert request.method == "POST"
+
+    
+    return redirect("hc-checks")
+
+@login_required
+def create_shopify_alerts(request):
+    assert request.method == "POST"
+
+    form = ShopifyForm(request.POST)
+    if form.is_valid():        
+        topic = form.cleaned_data["event"]        
+        API_KEY = form.cleaned_data["api_key"]
+        PASSWORD = form.cleaned_data["password"]
+        SHOP_NAME= form.cleaned_data['shop_name']
```

Comment:
Since these are not global variables, I don't think there is a need to capitalise them